cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-11-09 13:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(null=True, upload_to='movie_image_path'),

```

Comment:
Ensure that the `upload_to` parameter in `ImageField` is correctly set to a function or string that defines the path where images will be stored. If 'movie_image_path' is a function, make sure it is defined elsewhere in your codebase.